.github/workflows/code-style.yaml

```diff
@@ -0,0 +1,29 @@
+# Run code style checks on each pull request.
+name: Code style checks
+
+on: [pull_request]
+
+jobs:
+  linting:
+    env:
+      # Configure a constant location for the uv cache
+      UV_CACHE_DIR: /tmp/.uv-cache
+
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup uv
```

Comment:
What's the motivation for involving both uv and pixi here?

Reply:
Good catch, this was a copy/paste from LlamaBot, and I think I completely forgot that `pixi` can get you Python globally as well when I first wrote it. Thanks for the catch!